src/resources/Conversion/GeneralPartnershipResource.ts

```diff
@@ -25,9 +24,8 @@ export const GeneralPartnershipResource: ResourceIF = {
       orgTypesLabel: 'Business or Corporation',
       subtitle: 'You must have a minimum of two partners. You can add or remove partners (individual person or ' +
         'business) as well as change the mailing and delivery addresses and email address of individual people and ' +
-        'business partners.',
-      helpSection: null
```

Comment:
`helpSection` is not used and is optional, so I just deleted it from here (and some other files below).